lib/models/release.ts

```diff
@@ -410,9 +461,19 @@ const getReleaseModel = function (
 	 * 	console.log('finalized!');
 	 * });
 	 *
+	 * @example
+	 * balena.models.release.finalize({raw_version: '0.0.0', application: 456}).then(function(release) {
+	 * 	console.log('finalized!');
+	 * });
+	 *
 	 */
-	async function finalize(commitOrId: string | number): Promise<void> {
-		const { id } = await get(commitOrId, { $select: 'id' });
+	async function finalize(
+		commitOrIdOrRawVersion:
+			| string
+			| number
+			| { rawVersion: string; application: string | number },
+	): Promise<void> {
+		const { id } = await get(commitOrIdOrRawVersion, { $select: 'id' });
```

Comment:
I like this more, even if it's usually more risky. Let's try to be consistent and use this or the other way in both places:

lib/models/release.ts line 71